### PR TITLE
[Placeholder] Convert some graphOptz tests to placeholders

### PR DESCRIPTION
**Pull request by bertmaher**
*Description*: Start moving graphOptzTest to placeholders & context.

*Testing*: These tests.

*Documentation*: N/A

### CODE REVIEW

**Comment by bertmaher** [comment]
Haven't finished all these yet but wanted to make sure I'm on the right track.  I'm assuming that even though these tests aren't really "executable" that we still want to use placeholder/context, for API consistency, right?

File: tests/unittests/graphOptzTest.cpp [hunk 1]
```diff
@@ -115,11 +117,12 @@ TEST_F(GraphOptz, liveCodeNotEliminated) {
 }
```
Comment (qcolombet) [inline tests/unittests/graphOptzTest.cpp]: Note: the test is failing because getVars don't include the placeholders.

**Comment by rdzhabarov** [comment]
`check format` seems to fail here :)

File: lib/Graph/Graph.cpp [hunk 2]
```diff
@@ -1923,6 +1923,34 @@ void Function::createLSTM(llvm::StringRef namePrefix,
...
+  // Figure out how many channels are in the tensor.
+  size_t channels = input.dims()[channelIdx];
+
+  // Allocate the learnable parameters beta and gamma.
+  auto *beta = getParent()->createPlaceholder(
+      ElemKind::FloatTy, {channels}, "beta", true);
+  ctx.allocate(beta)->init(glow::Tensor::InitKind::Zero, 0, getPRNG());
+
+  auto *gamma = getParent()->createPlaceholder(
+      ElemKind::FloatTy, {channels}, "gamma", true);
```
Comment (rdzhabarov) [inline lib/Graph/Graph.cpp]: curious, why this is always trainable?

Comment (bertmaher) [reply lib/Graph/Graph.cpp]: Hm, I was just copying this from the existing (non-Context) createBatchNorm.  Should it be otherwise?

Comment (nadavrot) [reply lib/Graph/Graph.cpp]: I think that this is correct. Instead of creating trainable variables we create trainable placeholders and initialize them.

**Pull request merged**